index/segment/fst/README.md

```diff
@@ -0,0 +1,45 @@
+FST Segment
+===========
+
+```
+
+┌───────────────────────────────┐           ┌───────────────────────────────┐
```

Comment:
Any reason to not use a single FST for the fields and terms using a non-UTF8 byte as the separator?

Reply:
talked offline: vellum's regexp parser doesn't support non-utf8 chars. example here -https://gist.github.com/prateek/fb2cb1a197d4dcfeaf39640a40af3787